NEMO/context_processors.py

```diff
@@ -85,11 +86,12 @@ def base_context(request):
         "date_input_js_format": date_input_js_format,
         "datetime_input_js_format": datetime_input_js_format,
         "no_header": request.session.get("no_header", False),
-        "safety_menu_item": customization_values.get("safety_main_menu") == "enabled",
-        "calendar_page_title": customization_values.get("calendar_page_title"),
-        "tool_control_page_title": customization_values.get("tool_control_page_title"),
-        "status_dashboard_page_title": customization_values.get("status_dashboard_page_title"),
-        "requests_page_title": customization_values.get("requests_page_title"),
-        "safety_page_title": customization_values.get("safety_page_title"),
+        "safety_menu_item": customization_values.get("safety_main_menu", "enabled") == "enabled",
+        "calendar_page_title": customization_values.get("calendar_page_title", "Calendar"),
+        "tool_control_page_title": customization_values.get("tool_control_page_title", "Tool control"),
+        "status_dashboard_page_title": customization_values.get("status_dashboard_page_title", "Status dashboard"),
+        "requests_page_title": customization_values.get("requests_page_title", "Requests"),
+        "safety_page_title": customization_values.get("safety_page_title", "Safety"),
+        "calendar_first_day_of_week": CalendarCustomization.get("calendar_first_day_of_week"),
```

Comment:
Just grab it directly from the list of customization values, to avoid a unnecessary database query.
```suggestion
        "calendar_first_day_of_week": customization_values.get("calendar_first_day_of_week"),
```